src/Cimpress.Auth0.Client/Auth0TokenProvider.cs

```diff
@@ -1,88 +1,73 @@
 ﻿using System;
-using System.Collections.Concurrent;
 using System.Collections.Generic;
 using System.Net.Http;
 using System.Net.Http.Headers;
-using System.Text.RegularExpressions;
-using System.Threading;
 using System.Threading.Tasks;
 using Cimpress.Auth0.Client.Proxies;
 using Microsoft.Extensions.Logging;
 
 namespace Cimpress.Auth0.Client
 {
-    public class Auth0TokenProvider : IAuth0TokenProvider
+    public class Auth0TokenProvider : BaseAuth0TokenProvider, IAuth0TokenProvider
```

Comment:
Good catch in that autoscheduler is passed but actually not used in the base class. I have removed that. 

Warren and I chatted a bit about refactoring for an injected class instead of a base class here. For now, we'll leave the base functionality and potentially change it later on.